jenkins/Test.groovy.j2

```diff
@@ -266,8 +266,13 @@ stage('Test') {
             )
           }
           {{ m.upload_artifacts(tag='docs', filenames=["docs.tgz"]) }}
+          def docs_path = "tvm-jenkins-artifacts-prod/tvm/${env.BRANCH_NAME}/docs/"
           sh(
-            script: "aws s3 cp --no-progress _docs s3://${s3_prefix}/docs --recursive",
+            script: """
+              set -eux
+              aws s3 rm --no-progress s3://${docs_path} --recursive
```

Comment:
I think we should keep the uploads in separate iterations so old ones are still accessible. This complicates this bot a bit since it can't just post the comment once, it needs to check on every time it gets a status and update the current link (ideally we wouldn't leave more comments but search for the old one and update it in place), but the URL could then be something like `https://d3f1x5ne0bf47p.cloudfront.net/tvm/PR-1/2/docs/index.html`

Reply:
Do you have an example of where we've achieved this before?